common/common-framework/src/test/java/cn/iocoder/common/framework/util/DateUtilTest.java

```diff
@@ -0,0 +1,56 @@
+package cn.iocoder.common.framework.util;
+
+import org.junit.Test;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class DateUtilTest {
+
+    @Test
+    public void testAddDate() {
+        assertNull(DateUtil.addDate(0, 0));
+        assertEquals(1_778_410_800_000L, DateUtil.addDate(
+                new Date(1_515_585_600_000L), 2, 100).getTime());
+    }
+
+    @Test
+    public void testFormat() {
+        assertEquals("", DateUtil.format(null, null));
+        assertEquals("2018-01-10:12:00:00", DateUtil.format(
+                new Date(1_515_585_600_000L), "yyyy-MM-dd:HH:mm:ss"));
+    }
+
+    @Test
+    public void testGetDayBegin() {
+        assertNull(DateUtil.getDayBegin(null));
+        assertEquals(1_515_542_400_000L, DateUtil.getDayBegin(
+                new Date(1_515_585_600_000L)).getTime());
```

Comment:
do the same as for `testAddDate` (wrap `expected` with `new Date(` and remove `.getTime()`)